### PR TITLE
correct grammar typo; explain what generates the recommended rules file

### PR DESCRIPTION
**Pull request by caseywatts**

### CODE REVIEW

File: scripts/update-rules.js [hunk 1]
```diff
@@ -76,8 +76,9 @@ const recommendedRules = rules.reduce((obj, entry) => {
 
 const recommendedRulesContent = `/*
  * IMPORTANT!
- * This file has been automatically generated,
- * in order to update it's content execute "npm run update"
+ * This file has been automatically generated from ,
```
Comment (michalsnik) [inline scripts/update-rules.js]: from what? :D

File: scripts/update-rules.js [hunk 2]
```diff
@@ -76,8 +76,9 @@ const recommendedRules = rules.reduce((obj, entry) => {
...
+ * in order to update its content based on the rule definition
```
Comment (michalsnik) [inline scripts/update-rules.js]: Maybe: `in order to update its content based on rules' definitions, execute "npm run update"`?

**Comment by caseywatts** [comment]
updated based on @michalsnik 's feedback - thanks! :)

**Comment by michalsnik** [comment]
Thanks @caseywatts :)

**Pull request merged**